README.md

```diff
@@ -26,12 +26,31 @@ All profiles use DNS over HTTPS and support both IPv4 and IPv6.
 8. You'll see a warning that the profile might monitor your network traffic, and that it's unsigned. Tap **Install**.
 9. The profile will be installed. Tap **Done**.
 
+## Install on macOS 11 or higher
+
+1. Using Safari, choose the desired profile above.
+2. Right-click the **Raw** button and choose *Download Linked File*.
+3. Locate the downloaded file and remove the *.txt* file extension (so it ends in *.mobileconfig* only).
```

Comment:
Nitpicky, but can you use a code block `like this` instead of italics for *.txt* and *.mobileconfig*?

Reply:
addresses below, please use "correctly resolve feedback, lol" lol